conf/messages

```diff
@@ -219,7 +219,7 @@ error.required.ba.involved.in.other.text = Enter the other activities your busin
 error.invalid.ba.involved.in.other.text = Other activities must be 255 characters or less
 error.invalid.characters.ba.involved.in.other.text = Other activities can include letters a to z, spaces, and most punctuation and special characters
 error.required.ba.business.turnover = Select how much total net profit your business expects in the next 12 months
-error.required.ba.turnover.from.mlr = Select how much total net profit your business expects in the next 12 months from the services you’re registering
+error.required.ba.turnover.from.mlr = Select how much total net profit your business expects in the next 12 months from the services you're registering
```

Comment:
Replace ' with ’ or it will not show in the string.

Reply:
Done all :+1: good spot